### PR TITLE
Fix \Ds\Map::values() example

### PR DESCRIPTION
**Pull request by enumag**

### CODE REVIEW

File: reference/ds/ds/map/values.xml [hunk 1]
```diff
@@ -46,12 +46,12 @@ var_dump($map->values());
    &example.outputs.similar;
    <screen>
 <![CDATA[
-object(Ds\Map)#2 (3) {
-  [0]=>
+class Ds\Vector#2 (3) {
```
Comment (nikic) [inline reference/ds/ds/map/values.xml]: This does not look like a standard var_dump. Maybe you have XDebug enabled?

**Comment by cmb69** [comment]
DS 1.2.9 produces for me:
````
object(Ds\Vector)#2 (3) {
  [0]=>
  int(1)
  [1]=>
  int(2)
  [2]=>
  int(3)
}
````
So just the class name would have to fixed in the docs.

**Comment by nikic** [comment]
@php-pulls merge

**Comment by enumag** [comment]
@nikic You're right, it was because of xdebug. I had no idea it changed var_dump format.

**Pull request closed without merging**